packages/service-clients/odsp-client/package.json

```
@@ -48,13 +48,16 @@
 	"types": "lib/public.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "flub generate entrypoints --outDir ./dist",
-		"api-extractor:esnext": "flub generate entrypoints --outDir ./lib --node10TypeCompat",
 		"build": "fluid-build . --task build",
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "api-extractor run --local",
 		"build:esnext": "tsc --project ./tsconfig.json",
+		"build:node10:entrypoints": "flub generate entrypoints --outDir ./lib/entrypoints --node10TypeCompat",
+		"build:entrypoints": "flub generate source-entrypoints",
+		"build:entrypoints:esnext:cjs": "npm run build:entrypoints:esnext && npm run build:entrypoints:cjs",
+		"build:entrypoints:cjs": "tsc --project ./src/entrypoints/tsconfig.cjs.json",
+		"build:entrypoints:esnext": "tsc --project ./src/entrypoints/tsconfig.json",
```

If all of these are needed, I suggest the following naming convention to be more consistent with what we do in other tasks.

```suggestion
		"generate:entrypoints": "flub generate sourceEntrypoints",
		"build:entrypoints": "concurrently npm:build:entrypoints:esnext npm:build:entrypoints:cjs && npm run build:entrypoints:node10",
		"build:entrypoints:cjs": "fluid-tsc --project ./src/entrypoints/tsconfig.cjs.json",
		"build:entrypoints:esnext": "tsc --project ./src/entrypoints/tsconfig.json",
		"build:entrypoints:node10": "flub generate entrypoints --outDir ./lib/entrypoints --node10TypeCompat",
```

Then the fluid-build task definitions would be:

```
	"generate:entrypoints": ["build:esnext", "commonjs"], // assuming we have some CJS-only packages
	"build:source:entrypoints": ["build:esnext"],
	"build:entrypoints:cjs": ["generate:entrypoints", "commonjs"],
	"build:entrypoints:esnext": ["generate:entrypoints"],
	"build:node10:entrypoints": ["build:entrypoints:esnext"],
	"build:entrypoints": ["generate:entrypoints"],
```